lang/cs/Org.Apache.REEF.Driver/Bridge/Events/FailedTask.cs

```diff
@@ -33,7 +32,18 @@ internal sealed class FailedTask : IFailedTask
         
         public FailedTask(IFailedTaskClr2Java failedTaskClr2Java)
         {
-            Parse(failedTaskClr2Java);
+            var serializedInfo = failedTaskClr2Java.GetFailedTaskSerializedAvro();
+            var avroFailedTask = AvroJsonSerializer<AvroFailedTask>.FromBytes(serializedInfo);
+
+            Id = avroFailedTask.identifier;
```

Comment:
This is defined in `IIdentifiable`. The class hierarchy is such `IIdentifiable->IAbstractFailure->IFailedTask->FailedTask`.